src/OrchardCore.Themes/TheBlogTheme/Views/Layout.liquid

```diff
@@ -2,37 +2,37 @@
 <html>
 <head>
     <title>{% page_title Site.SiteName, position: "before", separator: "-" %}</title>
-    {% helper "meta", charset:"utf-8" %}
-    {% helper "meta", http_equiv:"X-UA-Compatible", content:"IE=edge" %}
-    {% helper "meta", name:"viewport", content:"width=device-width, initial-scale=1.0" %}
-    {% helper "link", type:"image/x-icon", rel:"shortcut icon", src:"/TheBlogTheme/favicon.ico" %}
+    {% meta charset:"utf-8" %}
+    {% meta http_equiv:"X-UA-Compatible", content:"IE=edge" %}
+    {% meta name:"viewport", content:"width=device-width, initial-scale=1.0" %}
+    {% link type:"image/x-icon", rel:"shortcut icon", src:"/TheBlogTheme/favicon.ico" %}
 
-    {% helper "script", name:"jquery", use_cdn:"true", at:"Foot" %}
+    {% script name:"jquery", use_cdn:"true", at:"Foot" %}
 
     {% helperblock "script", at:"Foot" %}
         var foo = "bar";
     {% endhelperblock %}
 
-    {% helper "script", name:"bootstrap", use_cdn:"true", at:"Foot" %}
+    {% script name:"bootstrap", use_cdn:"true", at:"Foot" %}
     <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
-    {% helper "script", src:"/TheBlogTheme/Scripts/clean-blog.min.js", debug_src:"/TheBlogTheme/Scripts/clean-blog.js", at:"Foot" %}
+    {% script src:"/TheBlogTheme/Scripts/clean-blog.min.js", debug_src:"/TheBlogTheme/Scripts/clean-blog.js", at:"Foot" %}
 
-    {% helper "link", rel:"stylesheet", src:"/TheBlogTheme/Styles/blog.min.css" %}
+    {% link rel:"stylesheet", src:"/TheBlogTheme/Styles/blog.min.css" %}
 
     <!-- Custom Fonts -->
-    {% helper "style", name:"font-awesome", use_cdn:"true" %}
-    {% helper "link", src:'https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic', rel:'stylesheet', type:'text/css' %}
-    {% helper "link", src:'https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800', rel:'stylesheet', type:'text/css' %}
+    {% style name:"font-awesome", use_cdn:"true" %}
+    {% link src:'https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic', rel:'stylesheet', type:'text/css' %}
```

Comment:
Doesn't look like it needs a liquid tag here, just a plain old html one. If it was in another shape than the layout it would have been fine.